modules/core/src/views/view-manager.js

```diff
@@ -81,7 +81,11 @@ export default class ViewManager {
 
   // Get a set of viewports for a given width and height
   // TODO - Intention is for deck.gl to autodeduce width and height and drop the need for props
-  getViewports() {
+  // @param rect ([object]) - x, y, [width], [height] of the pixel/area that the viewports should cover
```

Comment:
Nit: Update the comment, to reflect `rect` is optional, and returns all viewports when not provided.